tests/agents/test_orchestrator.py

```
@@ -0,0 +1,366 @@
+
+        slow_agent.execute = slow_execute
+
+        if hasattr(orchestrator, 'execute_task'):
+            task = {
+                "id": "task-slow",
+                "agent_id": slow_agent.id,
+                "timeout": 1  # 1 second timeout
+            }
+
+            with patch.object(orchestrator, 'get_agent', return_value=slow_agent):
+                with pytest.raises(asyncio.TimeoutError) or result:
```

Flake8 (7.3.0)

[error] undefined name 'result'

(F821)